### ISSUE TITLE
Business cards | Front

### ISSUE THREAD

**Issue by iteles** [dwyl/print-me#7]
Related to issue #6 on the back design of the business cards.

**Comment by iteles** [comment]
The option that's not here of course is plain white on the back with just the lettering and a small dwyl icon next to the name.

![dwyl-business-card -back-portrait-plain border](https://cloud.githubusercontent.com/assets/4185328/8310568/afabc250-19c8-11e5-9840-3c566a6fa711.png)
![dwyl-business-card -back-portrait-slogan border](https://cloud.githubusercontent.com/assets/4185328/8310570/b047acce-19c8-11e5-8473-1dd9ea44aaca.png)
![dwyl-business-card -back-landscape border](https://cloud.githubusercontent.com/assets/4185328/8310567/afa5a0a0-19c8-11e5-8daf-6081f0f2a503.png)
![dwyl-business-card -back-landscape-with-icons border](https://cloud.githubusercontent.com/assets/4185328/8310569/afbf237c-19c8-11e5-8de6-2ad7718c9963.png)

**Comment by iteles** [comment]
@nelsonic I realise the twitter and the internet icons are the wrong way round :see_no_evil: but please let me know if you like any of these or would just like the back to be plain. They need to be at the printers tomorrow (Wednesday 24th) morning.

**Comment by nelsonic** [comment]
Cutting off the logo is _not_ an option. I'm "_not fussed_" what you do with the rest. this is an _Alpha_ business card.

**Comment by nelsonic** [comment]
Any reason why you cannot re-use the design from the T-Shirts?

**Comment by iteles** [comment]
This is the _back_, having anything other than just writing here is just to add visual interest. The front is where we can just re-use the t-shirt design.

**Comment by nelsonic** [comment]
I don't _need_ my card to be "_visually interesting_" I need it to be plain but informative.

**Comment by iteles** [comment]
<img width="993" alt="business-card-back-sample" src="https://cloud.githubusercontent.com/assets/4185328/10699313/b7e2a20a-79ad-11e5-88f6-2da1d11e0679.png">

**Comment by iteles** [comment]
@nelsonic  No border on the actual card, this is just to give an idea of whitespace:

<img width="1004" alt="business-card-back-sample" src="https://cloud.githubusercontent.com/assets/4185328/10699341/e1487c6e-79ad-11e5-9d58-e8e3f934c556.png">

**Comment by nelsonic** [comment]
@iteles we can test this text on the landing page. Its too wordy and not enough of a draw when printed on a card.

**Comment by nelsonic** [comment]
Instead I would write a **_question**_ on the card:

> Do you ever get to the end of a day and wonder where the time went?

or

> How do you know what the most important thing is to work on?

see: http://www.amazon.com/Leading-Questions-Leaders-Solutions-Knowing/dp/1118658132/ref=dp_ob_title_bk

**Comment by iteles** [comment]
> Are you spending your time on your most important tasks?

I feel like any of these phrases don't work too well on business cards, they just seem to be in a tone different to the one we want to go for :worried:

**Comment by nelsonic** [comment]
Ok. please make a decision. thanks.

**Comment by iteles** [comment]
Getting **100 /websummit** of these made for **_each_** of us.

<img width="410" alt="dwyl-final-business-cards-front" src="https://cloud.githubusercontent.com/assets/4185328/10719185/b238e038-7b7b-11e5-9e62-8b64630d1315.png">

**Comment by nelsonic** [comment]
100...? is that how many people you are expecting to interact with? fair enough.

**Comment by iteles** [comment]
No, we'll have 200 business cards each, which will also give us an opportunity to trial the idea of sharpie-ing on routes. I can order more Nelson, just tell me how many you want with the _/websummit_ route please.

**Comment by nelsonic** [comment]
If you are happy with 200 cards, please place the order.

**Comment by iteles** [comment]
Alpha business cards done. Road testing these at the [Web Summit](https://websummit.net/) in a couple of days and gathering feedback on anything that confuses people or questions they ask.

![img_6318](https://cloud.githubusercontent.com/assets/4185328/10866587/63976c22-802d-11e5-81f0-faaeb4e1a144.jpg)

**Comment by nelsonic** [comment]
@markwilliamfirth what do you want *yours* to say?
